ckanext/scheming/validation.py

```diff
@@ -83,6 +88,27 @@ def validator(key, data, errors, context):
     return validator
 
 
+@scheming_validator
+def scheming_isodatetime(value, context):
```

Comment:
the parameters to scheming validators are the whole schema and the field schema, not value and context. It seems like you're not using these values at all, so don't bother with `@scheming_validator` at all